nav2_behavior_tree/plugins/action/clear_grid_around_pose_service.cpp

```diff
@@ -0,0 +1,30 @@
+#include <string>
+#include <memory>
```

Comment:
`<string>` is already included in the header file and I don't know if the `<memory>` needs to be included at all.

Reply:
Removed